code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -41,32 +56,107 @@
 			return -1
 	if (!src.listening)
 		return -1
-	if(freq in ANTAG_FREQS)
+	if(freq == SYND_FREQ)
 		if(!(src.syndie))
 			return -1//Prevents broadcast of messages over devices lacking the encryption
 
 	return canhear_range
 
 
-/obj/item/device/radio/intercom/hear_talk(mob/M as mob, msg)
-	if(!src.anyai && !(M in src.ai))
+/obj/item/device/radio/intercom/hear_talk(message, atom/movable/speaker, message_langs, raw_message, radio_freq)
+	if(!src.anyai && !(speaker in src.ai))

```

Comment:
May want to look at the ramifications of changing this. Reason being we have completely different saycode than VG's terribad saycode (which we killed off, entirely).